server/modules/analysis/service/_rChain/_files/system/init-api.R

```diff
@@ -0,0 +1,39 @@
+usePackage("httr");
```

Comment:
add this to init.r (or even better, let's create a init-packages.R where all the needed packages will go. there are a lot of them )

Reply:
ok